Dockerfile

```diff
@@ -0,0 +1,14 @@
+FROM golang:1.13-alpine as builder
+
+ARG DB_TYPE
+
+WORKDIR /build
+COPY . /build
+SHELL ["/bin/ash", "-o", "pipefail", "-c"]
+
+RUN apk --no-cache add make
+
+RUN DB_TYPE=${DB_TYPE} make db-all
```

Comment:
This line always overwrites `DB_TYPE`. `docker build . -t trivy-db` doesn't seem to work because an empty `DB_TYPE` is passed to `make`.
https://github.com/aquasecurity/trivy-db/pull/22/files#diff-04c6e90faac2675aa89e2176d2eec7d8R35

You can see the error. I think `./assets/.db` should be `./assets/trivy.db`.
```
mkdir -p assets
/go/bin/bbolt compact -o ./assets/.db cache/db/trivy.db
576761856 -> 126279680 bytes (gain=4.57x)
rm cache/db/trivy.db
make[1]: Leaving directory '/build'
make db-compress
make[1]: Entering directory '/build'
gzip assets/*
gzip: assets/*: No such file or directory
make[1]: *** [Makefile:87: db-compress] Error 1
make[1]: Leaving directory '/build'
make: *** [Makefile:63: db-all] Error 2
```

Reply:
Thanks, I'll set the default arg for DB_TYPE to trivy.